pkg/cluster/k8sres.go

```diff
@@ -1253,6 +1257,34 @@ func (c *Cluster) generateCloneEnvironment(description *acidv1.CloneDescription)
 	return result
 }
 
+func (c *Cluster) generateStandbyEnvironment(description *acidv1.StandbyDescription) []v1.EnvVar {
+	result := make([]v1.EnvVar, 0)
+
+	if description.S3WalPath == "" {
+		return result
+	}
+
+	cluster := description.ClusterName
+	result = append(result, v1.EnvVar{Name: "STANDBY_SCOPE", Value: cluster})
+
+	// standby with S3, find out the bucket to setup standby
+	msg := "Standby from S3 bucket"
+	c.logger.Info(msg, description.S3WalPath)
+
+	msg = "Use custom parsed S3WalPath %s from the manifest"
+	c.logger.Warningf(msg, description.S3WalPath)
+
+		result = append(result, v1.EnvVar{
+			Name:  "STANDBY_WALE_S3_PREFIX",
+			Value: description.S3WalPath,
+		})
+
+	result = append(result, v1.EnvVar{Name: "STANDBY_METHOD", Value: "STANDBY_WITH_WALE"})
```

Comment:
1. is there a plan/need to support  `STANDBY_WITH_WAL-G` ?
2. please document the env vars used here in [the Spilo environment doc ](https://github.com/zalando/spilo/blob/96813721e9b23cbe9d2511ffdf93db2a4d73f7ed/ENVIRONMENT.rst)

Reply:
where is implementation for these env vars? I can't find any mentions of it in https://github.com/zalando/spilo

Reply:
Here is the PR for doing the same --> https://github.com/zalando/spilo/pull/343 feel free to comment there.